egs/wsj/s5/utils/prepare_lang.sh

```diff
@@ -67,6 +67,10 @@ extra_word_disambig_syms=        # if set, add disambiguation symbols from this
 num_extra_phone_disambig_syms=1 # Standard one phone disambiguation symbol is used for optional silence.
                                 # Increasing this number does not harm, but is only useful if you later
                                 # want to introduce this labels to L_disambig.fst
+boost_unk=1.0 # for unknown-word modeling, we provide this option to boost the unknown-word's probability
```

Comment:
please call this unk_factor and rework the explanation.
and instead of 'improper', say, 'too low or too high'.